core/src/main/java/com/radixdlt/environment/RemoteEventProcessor.java

```diff
@@ -64,18 +64,17 @@
 
 package com.radixdlt.environment;
 
-import com.radixdlt.consensus.bft.BFTNode;
 import com.radixdlt.environment.rx.RemoteEvent;
 
 /**
  * Processes events from a remote node
  *
  * @param <T> the event class
  */
-public interface RemoteEventProcessor<T> {
-  void process(BFTNode sender, T t);
+public interface RemoteEventProcessor<N, T> {
```

Comment:
I'd really like us to explain what `N` is here and on the MessageTransportType interface, and explain what it means for the messages and their receivers.